hooks/pre/30-upgrade_postgresql.rb

```diff
@@ -0,0 +1,15 @@
+def postgresql_10_upgrade
+  execute('foreman-maintain service start --only=postgresql')
+  (name, owner, enconding, collate, ctype, privileges) = %x{runuser postgres -c 'psql -lt | grep -E "^\s+postgres"'}.chomp.split('|').map(&:strip)
+  execute('foreman-maintain service stop')
```

Comment:
Checking, @ekohl are you OK with this being here with a requirement on RPM foreman-installer packaging to require foreman-maintain? This code is gated on `el7?` so Debian is unaffected.

Reply:
While I'm not Ewoud, I'd be OK with that. But I think I'd prefer the file to be called `30-el7_upgrade_postgresql` to denote it's only relevant on el7.